eng/Directory.Build.Common.targets

```diff
@@ -147,12 +147,12 @@
     <Compile Include="$(MgmtCoreShared)/**/*.cs"
                  Link="Shared/Management/%(RecursiveDir)%(Filename)%(Extension)" />
   </ItemGroup>
-  
+
   <!--TODO: end-->
-  <!-- *********** Management Client Library Override section ************* -->
+  <!-- *********** Management Client Library Override section (all) ************* -->
   <ItemGroup Condition="'$(IsMgmtLibrary)' == 'true' and '$(IsTestProject)' != 'true'">
     <PackageReference Include="System.Text.Json" />
-    
+
```

Comment:
<PackageReference Condition="not core and not resources" Include="Azure.ResourceManager.Core" />

add a comment to the ado work item to remove "resources"